ramenctl/ramenctl/resources/configmap.yaml

```diff
@@ -38,6 +38,7 @@ data:
       disabled: $volsync_disabled
     multiNamespace:
       FeatureEnabled: true
+      volsyncSupported: true
```

Comment:
We should really use $volsync_enabled since ramenctl enabled volsync only if the env report the "volsync" feature:

```
ramen:
  hub: hub                                                                                          
  clusters: [dr1, dr2]
  topology: regional-dr
  features:
    volsync: true
```

For example the regional-dr.yaml have "volsync: false".

Reply:
Ok, lets keep it simple then.